### ISSUE TITLE
Windows Docker Config

### ISSUE THREAD

**Issue by 2wheelsdown** [ninthwalker/NowShowing#74]
Not a real 'issue', but I was directed to post my question here from the Plex sub-reddit: https://www.reddit.com/r/PleX/comments/8av2xe/nowshowing_v2_released/dx6eskb/

As per that comment on Reddit, I am a total newbie to the world of Docker.  I've managed to get 99% of the way there, but I'm unsure about how to 'volume map' the NowShowing container when I run it to give it a writable space to save its' data/config.

Thanks for any assistance offered.

**Comment by ninthwalker** [comment]
Hi again. 
As mentioned, you will need to map a local host volume to the internal docker directory ‘/config’. Otherwise settings and such won’t be saved on docker container wipes.

I am not super familiar with docker on Windows, but the same docker run command listed on the wiki should work. Just use your own windows path. 
ie:
-v “c:\dockermappings\nowshowing”:”/config”:”rw”

I think you may have to share out the C drive in your Docker settings on windows first though.
Depending on the version, try right clicking the system tray icon, go to settings, and then check out the shares section.

Let me know how that goes. 
Thanks.

**Comment by 2wheelsdown** [comment]
Yes!!  Awesome, thanks for the pointer.  I understand now what the volume mapping achieves.  I used the following command to start a new instance of the container:

docker run -d --name=NowShowingV2 --net=bridge -e TZ="Europe/London" -p 6878:6878/tcp -v "C:\DockerApps\NowShowingV2:/config:rw" ninthwalker/nowshowing:v2

And sure enough, a little window popped up saying "Docker is asking to share the C: drive - Share?", gave it the local user account password again and bingo.  The app is now running, I can peer into the mapped folder to see the files and config the app is saving, which is lucky too because it didn't like the "$" used in my admin password so I had to peer into the 'Secure.php' file to see what password it actually had saved so I could log in.

Now I can start playing around with this lovely looking app of yours.  Cheers for the assist.
